plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -389,16 +384,14 @@ public class DeltaLakeMetadata
     // The highest reader and writer versions Trino supports
     private static final int MAX_READER_VERSION = 3;
     public static final int MAX_WRITER_VERSION = 7;
-    // The lowest versions Delta Lake supports reader and writer features
-    private static final int MIN_READER_FEATURE_VERSION = 3;
-    private static final int MIN_WRITER_FEATURE_VERSION = 7;
-    private static final int CDF_SUPPORTED_WRITER_VERSION = 4;
-    private static final int COLUMN_MAPPING_MODE_SUPPORTED_READER_VERSION = 2;
-    private static final int COLUMN_MAPPING_MODE_SUPPORTED_WRITER_VERSION = 5;
-    private static final int TIMESTAMP_NTZ_SUPPORTED_READER_VERSION = 3;
-    private static final int TIMESTAMP_NTZ_SUPPORTED_WRITER_VERSION = 7;
-    private static final int DELETION_VECTORS_SUPPORTED_READER_VERSION = 3;
-    private static final int DELETION_VECTORS_SUPPORTED_WRITER_VERSION = 7;
+
+    public static final int CDF_SUPPORTED_WRITER_VERSION = 4;
```

Comment:
Maybe those should be moved to `DeltaLakeTableFeatures` to be in the same place as feature name and min version

Reply:
I would ignore this suggestion. This constant is mainly for `delta.enableChangeDataFeed` configuration, not table features.